public/app/features/logs/components/LogRow.tsx

```diff
@@ -121,6 +127,13 @@ class UnThemedLogRow extends PureComponent<Props, State> {
     }
   };
 
+  onMouseMove = (e: MouseEvent) => {
+    // The user is selecting text, so hide the log row menu so it doesn't interfere.
+    if (document.getSelection()?.toString() && e.buttons > 0) {
+      this.setState({ mouseIsOver: false });
+    }
```

Comment:
Nice!

Reply:
Just noticed in dashboards, that the copy button isn't reappearing, which makes sense, but just want to let that here. Maybe something we can debounce after a certain timeout:


https://github.com/grafana/grafana/assets/8092184/6bb1302f-7a5f-4a0f-87af-4c8ddb6ac47f

Reply:
Ohh, good catch!

Reply:
This has been fixed.